jetson/nav/.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
wait what are the .idea files?

Reply:
Oops, removed, must've been snuck in by CLion